wallet/tutorials/javascript-dapp-simple.md

```diff
@@ -0,0 +1,317 @@
+
+```jsx title="index.html"
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Simple dapp</title>
+</head>
+<body>
+    <script src="../dist/index.js"></script>  <!-- Links to the JavaScript bundle created by Webpack -->
+</body>
+</html>
+```
+
+### 2. Install and configure Webpack
+
+Navigate to the root of your `simple-dapp` directory and run the following commands to initialize npm and install Webpack.
+
+```bash 
+npm init -y && npm install --save-dev webpack webpack-cli
+```
+
+Create a `webpack.config.js` in the root of your project with the following configuration:
+
+```js title="webpack.config.js"
+const path = require("path");
+
+module.exports = {
+  entry: "./src/index.js",
+  output: {
+    path: path.resolve(__dirname, "dist"),
+    filename: "index.js",
+  },
+  mode: "development",
+};
+```
+
+Then, edit `package.json` to include a build script:
+
+```json title="package.json"
+"scripts": {
+  "build": "webpack"
+}
+```
+
+When you run `npm run build` from your terminal, Webpack bundles the content in `src/index.js` into `dist/main.js`, which is linked in `index.html`.
+
+### 3. Detect MetaMask
+
+:::caution
+
+Using the `window.ethereum` provider object to detect MetaMask is deprecated. We recommend [connecting to MetaMask using EIP-6963](../how-to/connect/index.md) for production environments.
+
+:::
+
+Install the `@metamask/detect-provider` module in your project directory:
+
+```bash
+npm i @metamask/detect-provider
+```
+
+Add the following code to `src/index.js` to detect the MetaMask provider using `@metamask/detect-provider`:
+
+```js title="index.js"
+import detectEthereumProvider from "@metamask/detect-provider";
+
+async function setup() {
+  const provider = await detectEthereumProvider();
+
+  if (provider && provider === window.ethereum) {
+    console.log("MetaMask is available!");
+    startApp(provider); // Initialize your dapp with MetaMask.
+  } else {
+    console.log("Please install MetaMask!");
+  }
+}
+
+function startApp(provider) {
+  if (provider !== window.ethereum) {
+    console.error("Do you have multiple wallets installed?");
+  }
+}
+
+window.addEventListener("load", setup);
+```
+
+#### Serve the project
+
+To test and verify your project works with MetaMask, serve your project through a local server. 
+If using VS Code and [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), right-click the `index.html` file, and select **Open with Live Server**. 
+ 
+Alternatively, you can install `http-server` using the following command:
+
+```bash
+npm install -g http-server
+```
+
+Then navigate to the project directory, and run `http-server`. 
+
+Open your browser and go to the provided local server URL (typically http://127.0.0.1:8080).
+
+### 4. Determine a user's connected network
+
+[Detect the user's network](#3-detect-metamask) to ensure all RPC requests are submitted to the currently connected network.
```

Comment:
```suggestion
[Detect the user's network](../how-to/manage-networks/detect-network.md) to ensure all RPC requests are submitted to the currently connected network.
```

This should link to the page explaining more about detecting a user's network, and not to the previous step that detects MetaMask.